### ISSUE TITLE
Rename `Learn` section

### ISSUE THREAD

**Issue by guimachiavelli** [meilisearch/documentation#1305]
Once #1304 is merged, we will have two main links in our top nav:

- Learn
- Reference

In combination with #1294, #1304 greatly increased the breadth of content under the `Learn` heading, to the point it is an awkward description that doesn't really give users a good understanding of what they'll find in that section.

We should rename it to something more descriptive. Let's gather some ideas here and discuss/vote on them later.

Current suggestions:
- Guides

**Comment by bidoubiwa** [comment]
I like `guides`, I think it is the most known and also very clear in what you are going to find inside

**Comment by dichotommy** [comment]
Some sites refer to their non-API documentation as just "docs", e.g. on Stripe's API reference, you click a button that says "Docs" to return to the main site.

I think it's not nearly as clear as "guides" but it does sort of get at the difference between the two. One is like a dictionary and the other one is more like a manual.

**Comment by bidoubiwa** [comment]
Yes I agree, sometimes I see docs as well and you are right it is quite confusing. Unless it is on their main website but then you are redirected to a docusaurus/vuepress like website in which everything becomes doc. 
Guide seems to be a good start while maybe not the perfect one yet

**Comment by maryamsulemani97** [comment]
I noticed that many websites with a "Guides" section have a "Quick Start"/"Getting Started" section as well. Examples:
- [Flutter](https://docs.flutter.dev/)
- [Gocardless](https://developer.gocardless.com/getting-started/api/introduction/)
- [Pandas](https://pandas.pydata.org/docs/user_guide/index.html)
- [Keras](https://keras.io/)
- [boxdev](https://developer.box.com/reference/)

Maybe we could go with this?

**Comment by guimachiavelli** [comment]
@dichotommy, I think this could a good and relatively simple to do. Perhaps something we can discuss in the next Docs restructure meeting?

**Comment by curquiza** [comment]
Discussed with @guimachiavelli 
We want to have more information about devhub before knowing what do to with this issue. We indeed want to rename the section but we need to know more about devhub to make a decision
